doc/gov/chapters/chapter01.rst

```diff
@@ -4,34 +4,45 @@ Introduction
+The Cloud iNfrastructure Telco Task Force (CNTT) was founded by AT&T, Vodafone, Verizon, Deutsche Telekom, Orange, and
+China Mobile. Soon thereafter, additional telco operator and vendor partner participants began to join the Task Force.
+CNTT reached its first major milestone when it gained sponsorship and support of the GSMA and Linux Foundation
+Networking in Summer 2019. As of June 2020, there were over thirty operators and partners (VNF suppliers, third-party
+integrators, hw/sw suppliers) in its member community and these numbers have continued to grow. CNTT was collaborating
+very closely with OPNFV and there were dependencies and overlap between the work of the two communities. In the
+beginning of 2021 CNTT and OPNFV merged under the name `Anuket <https://anuket.io/>`__ to leverage the synergies between
+the two projects.
+
+The Anuket project community is leading the industry in creating a common infrastructure reference platform in the form
+of reference model and reference architecture definitions to better support virtualised and containerised Network
+Functions for the Telecom industry as a whole. The Anuket community includes many open source development projects to
+create and support reference implementations, and develop tests for reference certification platforms.
+
+The Anuket project operates under a Technical Steering Committee (TSC) and governing rules documented in the `Anuket
+Charter <https://anuket.io/wp-content/uploads/sites/119/2021/07/Anuket-Charter_073021.pdf>`__ and in the Anuket Project
+Operations and Guidelines document.
 
 How to participate
 ------------------
 
-Participating in Anuket does not need any formal membership, and nothing to sign except the `CLA <https://anuket.io/cla/>`__. Participation is open to anyone, whether you are an employee of an LFN member company, supporter, or observer. By participating, you automatically accept the individual anti-trust policies of LFN and GSMA, the joint Terms of Reference of LFN and GSMA the LFN Code of Conduct, as well as the LFN Trademark policy.
-
--  `LFN Anti-Trust Policy <https://r.lfnetworking.org/Antitrust%20Slide.pdf>`__
-
--  `GSMA Anti-Trust Policy <https://github.com/cntt-n/CNTT/blob/master/Anti-Trust_Notice_GSMA_20190911.png>`__
-
--  `CNTT Terms of Reference (ToR) LFN \| GSMA <https://wiki.lfnetworking.org/display/LN/Terms+of+Reference?src=contextnavpagetreemode>`__
-
-   -  Note: If you are not self-employed, then you must ensure that your employer also agrees to the ToR before participating.
-
--  `Code of Conduct LFN <https://lfprojects.org/policies/code-of-conduct/>`__
+Participating in Anuket does not need any formal membership, and nothing to sign except the `CLA
+<https://anuket.io/cla/>`__. Participation is open to anyone, whether you are an employee of an LFN member company,
+supporter, or observer. By participating, you automatically accept the individual anti-trust policies of LFN and GSMA,
```

Comment:
are there LFN supporters or observers

Reply:
Hi @pgoyal01, I am bit unsure how to read your comment: do you mean that there are NO LFN supporters or observers? If so, how about:

"Participation is open to anyone, whether you are an employee of an LFN member company or an individual contributor."

Reply:
@gkunz Perfect

Reply:
@pgoyal01 fixed in https://github.com/cntt-n/CNTT/pull/3129/commits/25fe075e515ab8191b149117ca863591ea84e48a